frontend-opal/src/components/PortfolioContainer/Instruments/InstrumentsItem.js

```diff
@@ -67,7 +98,8 @@ InstrumentsItem.propTypes = {
     profileImg: PropTypes.any,
     amount: PropTypes.number,
     marketValue: PropTypes.number,
-    symbol: PropTypes.string
+    symbol: PropTypes.string,
+    originalPrice: PropTypes.number
     // unrlzedPLPercentage: PropTypes.string,
     // unrlzedPL: PropTypes.number
```

Comment:
@hondahmd I guess these are not needed anymore

Reply:
updated